Code/Tools/FontImporter/FontImporterApp.cpp

```diff
@@ -0,0 +1,145 @@
+#include <FontImporterPCH.h>
+#include <Core/Assets/AssetFileHeader.h>
+#include <FontImporterApp.h>
+#include <Foundation/IO/FileSystem/DeferredFileWriter.h>
+
+ezFontImporterApp::ezFontImporterApp()
+  : ezApplication("FontImporter")
+{
+}
+
+ezResult ezFontImporterApp::BeforeCoreSystemsStartup()
+{
+  ezStartup::AddApplicationTag("tool");
+  ezStartup::AddApplicationTag("fontimporter");
```

Comment:
not needed (application tags are a bit of a hack and unless you know that you need it, you don't need to add your own)